src/applications/ivc-champva/10-10D/containers/App.jsx

```diff
@@ -24,6 +25,16 @@ const breadcrumbList = [
 ];
 
 export default function App({ location, children }) {
+  useEffect(() => {
+    // Insert CSS to hide 'For example: January 19 2000' hint on memorable dates
+    // (can't be overridden by passing 'hint' to uiOptions):
+    addStyleToShadowDomOnPages(
+      [''],
+      ['va-memorable-date'],
+      '#dateHint {display: none}',
+    );
+  });
+
```

Comment:
This is a stopgap until removing the date hint is officially supported (see PR summary).

Reply:
Could the `va-date` month/year variation be used for this instead of hacking the memorable date component? https://design.va.gov/storybook/?path=/docs/components-va-date--docs#month-year

Reply:
I can check! As far as I know though, all of our dates require the full MM/DD/YYYY, so cutting out the day would be a problem.

Reply:
Ah! For some reason I thought this was added because you only needed to MM/YYYY. I must be wrong about that.

Reply:
Oh right - it's just here to make sure we hide the extra perma-hint. The fix for that is coming soon though, I'm told! Good to merge this then?